_data/4cc.yaml

```diff
@@ -188,7 +189,7 @@
   usage:
   - Boot Policy
 - code: lpnh
-  description: LocalPolicy Nonce Hash
+  description: Local Policy Nonce Hash
```

Comment:
LocalPolicy is stylized that way in usage

Reply:
apparently not for bputil

Reply:
Fair - depends on context (I might make this “Local Policy `LocalPolicy`) for search?